packages/bitcore-wallet-client/src/lib/key.ts

```diff
@@ -416,6 +416,7 @@ export class Key {
 
     if (password) $.shouldBeString(password, 'provide password');
 
+    // this._checkCoin(opts.coin);
```

Comment:
Is this supposed to be commented?

Reply:
Yes, or removed. Coins (constant) contains a limited coins/token, and the goal is to add any token/chain in the future.